test/k8sT/manifests/test-cli.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: foo
+  labels:
+    id: foo
+spec:
+  containers:
+  - name: netperf
+    image: docker.io/cilium/netperf:latest
```

Comment:
Just a small note that ideally we should have a specific tag and not use `latest`, otherwise Kubernetes will keep download this tag every time.

Reply:
I agree. For this image, we currently only have one other tag, [`v0.0.2`](https://hub.docker.com/r/cilium/netperf/tags), and it doesn't seem equivalent to `latest`.

More generally, it's far from the only place we do this across the test codebase:
```
$ git grep ":latest" test/
test/helpers/constants/images.go:       HttpdImage = "docker.io/cilium/demo-httpd:latest"
test/helpers/constants/images.go:       KafkaClientImage = "docker.io/cilium/kafkaclient2:latest"
test/helpers/constants/images.go:       ZookeeperImage = "docker.io/digitalwonderland/zookeeper:latest"
test/k8sT/Updates.go:   // 4 - upgrade cilium to `k8s1:5000/cilium/cilium-dev:latest`
test/k8sT/Updates.go:   // 9 - re install cilium:latest image for remaining tests.
test/k8sT/manifests/demo-named-port.yaml:        image: docker.io/cilium/demo-httpd:latest
test/k8sT/manifests/demo-named-port.yaml:        image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/demo-named-port.yaml:        image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/demo.yaml:        image: docker.io/cilium/demo-httpd:latest
test/k8sT/manifests/demo.yaml:        image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/demo.yaml:        image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/demo_ds.yaml:        image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/demo_ds_local.yaml:        image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/externalIPs/apps/app2.yaml:        image: docker.io/cilium/demo-httpd:latest
test/k8sT/manifests/externalIPs/apps/app3.yaml:        image: docker.io/cilium/demo-httpd:latest
test/k8sT/manifests/external_pod.yaml:    image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/http-clients.yaml:        image: quay.io/isovalent/wrk2:latest
test/k8sT/manifests/http-deployment.yaml:        image: nginx:latest
test/k8sT/manifests/kafka-sw-app.yaml:        image: docker.io/spotify/kafkaproxy:latest
test/k8sT/manifests/kafka-sw-app.yaml:        image: docker.io/cilium/kafkaclient:latest
test/k8sT/manifests/kafka-sw-app.yaml:        image: docker.io/cilium/kafkaclient:latest
test/k8sT/manifests/kafka-sw-app.yaml:        image: docker.io/cilium/kafkaclient:latest
test/k8sT/manifests/kafka-sw-app.yaml:        image: docker.io/cilium/kafkaclient:latest
test/k8sT/manifests/lrp-test.yaml:    image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/lrp-test.yaml:    image: docker.io/cilium/demo-client:latest
test/k8sT/manifests/netcat-ds.yaml:        image: docker.io/cilium/demo-client:latest
test/provision/compile.sh:        docker rmi cilium/cilium:latest
test/provision/compile.sh:        docker rmi "${CILIUM_OPERATOR_TAG}:latest"
test/provision/compile.sh:        docker rmi "${CILIUM_OPERATOR_AWS_TAG}:latest"
test/provision/compile.sh:        docker rmi "${CILIUM_OPERATOR_AZURE_TAG}:latest"
test/provision/compile.sh:        docker rmi "${CILIUM_OPERATOR_GENERIC_TAG}:latest"
test/provision/compile.sh:        docker rmi "${HUBBLE_RELAY_TAG}:latest"
test/provision/externalworkload_install.sh:CLUSTER_ADDR=192.168.36.11:32379 HOST_IP=192.168.36.10 CILIUM_IMAGE=cilium/cilium:latest contrib/k8s/install-external-workload.sh
test/provision/k8s_install.sh:docker run --net outside --ip 192.168.9.10 --restart=always -d docker.io/cilium/demo-httpd:latest
test/provision/k8s_install.sh:docker run --net outside --ip 192.168.9.11 --restart=always -d docker.io/cilium/demo-httpd:latest
```
I'm happy to send a follow up PR to create e.g. `1.0` tags and use them across our tests, if we think that's the right way to proceed.

Reply:
Sounds good.